src/py21cmfast/wrapper.py

```diff
@@ -2738,6 +2746,16 @@ def run_lightcone(
         spin_temp_files = []
         ionize_files = []
         brightness_files = []
+
+        if (
+            rotation_cubes
+            and user_params.OUTPUT_ALL_VEL is False
+            and ("velocity" in lightcone_quantities)
```

Comment:
this line is probably not necessary. If you woud llke to be more secure, perhaps check if vx and vy are in the LC